### PR TITLE
Update network overview documentation

### PR DESCRIPTION
**Pull request by sysvinit**
This updates the network overview page in the platform documentation to better reflect the current state of reality and the differences in physical organisation between RZOB and WHQ.

### CODE REVIEW

**Review by ctheune — CHANGES_REQUESTED** [review]
Looks good, a few more changes needed as discussed and annotated.

File: src/infrastructure/networking/networking.md [hunk 1]
```diff
@@ -11,16 +11,22 @@
 
 ## Physical networks
 
-The physical network in our public datacenter is implemented using physically
-separated switches per layer 2 segment (VLAN) with a hot-spare for each switch
-type.
+The physical network in our public datacenter RZOB is implemented using
+physically separated switches per layer 2 segment (VLAN) with a hot-spare for
+each switch type. In WHQ, we operate multiple layer 2 segments using the same
+physical switches, using VLAN's within the switches for logical separation, but
```
[the remark this answers is not in the document]
Comment (sysvinit) [reply src/infrastructure/networking/networking.md]: See [8dcb119](https://github.com/flyingcircusio/doc/pull/30/commits/8dcb119e0b50f1cab0044f0372f2581c6b831557).

File: src/infrastructure/networking/networking.md [hunk 2]
```diff
@@ -11,16 +11,22 @@
...
+*without* hot-spares.
```
[the remark this answers is not in the document]
Comment (sysvinit) [reply src/infrastructure/networking/networking.md]: As above.

File: src/infrastructure/networking/networking.md [hunk 3]
```diff
@@ -11,16 +11,22 @@
...
-where as the storage networks are connected with 10G ports. The datacenter
-uplink from our routers uses redundant 2x10G connectivity.
+whereas the storage networks are connected with 10G links. In RZOB our routers
+have redundant 2x10G connectivity to our uplink provider. In WHQ our routers
+have a redundant 2x1G link to our uplink provider, however this provider only
+has *non-redundant* 2.5G connectivity itself.
 
-Routers are attached to upstream ports provided by the data center, a dedicated
-management link, and a link with tagged VLANs for everything else.
+In all locations, routers are connected to upstream ports provided by the
```
[the remark this answers is not in the document]
Comment (sysvinit) [reply src/infrastructure/networking/networking.md]: How do the changes in [8dcb119](https://github.com/flyingcircusio/doc/pull/30/commits/8dcb119e0b50f1cab0044f0372f2581c6b831557) look?

**Pull request merged**